pom.xml

```diff
@@ -61,7 +61,7 @@
     <revision>4.11.5</revision>
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.303.3</jenkins.version>
+    <jenkins.version>2.319.3</jenkins.version>
```

Comment:
Update https://github.com/jenkinsci/git-plugin/blob/d7159dc8566217e0e161fb67c8771585ade67f95/pom.xml#L78 accordingly.

Reply:
Ok i updated it, I hope correctly